publish_reports/rm.py

```
@@ -0,0 +1,43 @@
+import json
+import tqdm
+
+reports_dir = os.path.join(os.path.dirname(__file__), 'reports')
+
+print(f"Removing files from {reports_dir}")
+
+user_to_del = "ابراهيم أحمد2025"
+
+all_files = []
+
+for root, dirs, files in os.walk(reports_dir):
```

Ruff (0.8.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)